content/docs/reference/cmctl.md

```diff
@@ -34,20 +34,26 @@ This will also install shell completion.
 
 ### Manual Installation
 
-You need the `cmctl.tar.gz` file for the platform you're using, these can be
+You need the `cmctl` file for the platform you're using, these can be
 found on our
-[GitHub releases page](https://github.com/cert-manager/cert-manager/releases).
+[cmctl GitHub releases page](https://github.com/cert-manager/cmctl/releases).
 In order to use `cmctl` you need its binary to be accessible under
 the name `cmctl` in your `$PATH`.
 Run the following commands to set up the CLI. Replace OS and ARCH with your
 systems equivalents:
 
 ```console
-OS=$(go env GOOS); ARCH=$(go env GOARCH); curl -fsSL -o cmctl.tar.gz https://github.com/cert-manager/cert-manager/releases/latest/download/cmctl-$OS-$ARCH.tar.gz
-tar xzf cmctl.tar.gz
+OS=$(go env GOOS); ARCH=$(go env GOARCH); curl -fsSL -o cmctl https://github.com/cert-manager/cmctl/releases/latest/download/cmctl_${OS}_${ARCH}
+chmod +x cmctl
 sudo mv cmctl /usr/local/bin
```

Comment:
`install` combines `chmod` + `mv`:

```suggestion
sudo install cmctl /usr/local/bin
```

Reply:
how universal is this command (linux, osx ...)?